rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -127,3 +127,9 @@ repo_build_pip_extra_indexes:
 # variable to 'true'.
 # Docs: http://docs.openstack.org/developer/openstack-ansible-security/
 # apply_security_hardening: true
+
+# Based on https://bugs.launchpad.net/neutron/+bug/1590845
```

Comment:
I think it's preferable to reference the RPCO issue instead of this neutron one, the RPCO issue states that this was one of a number or issues affecting l3ha and so I wouldn't want the impression to be given that once 1590845 is fixed it can be re-enabled.